src/cloudant/design_document.py

```diff
@@ -48,6 +48,44 @@ def __init__(self, database, document_id=None):
         self.setdefault('indexes', dict())
 
     @property
+    def rewrites(self):
+        """
+        Provides an accessor property to a list of dictionaries with rewrite
+        rules in the locally cached DesignDocument.  Each rule for URL rewriting
+        is a JSON object with four fields: ``from``, ``to``, ``method``,
+        and ``query``.
+
+        Note: Requests that match the rewrite rules must have a URL path that
+        starts with ``/$DATABASE/_design/doc/_rewrite``.
+
+        Rewrite rule example:
+
+        .. code-block:: python
+
+            # Add the rule to ``rewrites`` and save the design document
+            ddoc = DesignDocument(self.db, '_design/ddoc001')
+            ddoc['rewrites'] = [
+                {"from": "/old/topic",
+                 "to": "/new/",
+                 "method": "GET",
+                 "query": {}
+                 }
+            ]
+            ddoc.save()
+
+        Once the rewrite rule is saved to the remote database, the GET
+        request URL ``/$DATABASE/_design/doc/_rewrite/old/topic?k=v`` would be
+        rewritten as ``/$DATABASE/_design/doc/_rewrite/new/k=v``.
```

Comment:
I appreciate that this sample is straight from the Cloudant docs, but is it actually correct? Why would the k=v parameters become part of the path?
Shouldn't it be:
`/$DATABASE/_design/doc/_rewrite/old/topic?k=v`
`/$DATABASE/_design/doc/_rewrite/new/?k=v` (which admittedly looks wrong because of the trailing / in `/new/`). In fact some of the couch examples show trailing slashes being omitted, so maybe it will just be rewritten as:
`/$DATABASE/_design/doc/_rewrite/new?k=v`